### ISSUE TITLE
first time with docker

### ISSUE THREAD

**Issue by tamatoa** [steamcache/generic#27]
Hello,
I dont have any experience with docker and steamcache.
After reading all differents threats, I install docker and start to pull steamcache/generic + steamcache-dns on my ubuntu cache server.

I type this command and hit enter:
docker run --restart=always --name cache-steam -v /cache/steam/data:/data/cache -v /cache/steam/logs:/data/logs -p 192.168.1.104:80:80 steamcache/generic:latest

I got the prompt:
nginx: the configuration file /etc/nginx.nginx.conf syntax is ok
nginx: configuration file /etc/nginx/nginx.conf test is sucessful
after the last line, I'm stuck inside the prompt. I tried to hit Ctrl + P, Ctrl + Q, Ctrl PQ, Ctrl + C, i'm still stuck inside. any idea how to deal with it ?

If someone had a white page how to proceed in every step pls.

THank you

**Comment by tiramon** [comment]
try to add -d to the call it should start detached then

https://docs.docker.com/engine/reference/run/#detached--d

**Comment by tamatoa** [comment]
it still doesnt work. So everytime I'm stuck I restart that ubuntu server and try differents commands to find out. did you have any experience by setting up a steamcache/generic + steamcache-dns on the same ubuntu machine ? Thanks

**Comment by JasonRivers** [comment]
@tamatoa could you be slightly more specific with "it doesn't work"? Are you still getting a locked terminal, or are you unable to communicate with the steamcache?

There's no reason to reboot the server, simply log in again from another terminal, and run `docker ps` This will show you the running containers.

We run both on the same server at LANs without any issues, the -d suggested by @tiramon should disconnect the container from your terminal so that you can run other commands, I don't see in your report that you're running the steamcache-dns container at all, You will need to run the steamcache container, DNS container and you'll also need to point your network DNS to the DNS container for steam to know where to find the cache.

If you are unfamiliar with docker, I suggest you have a quick read of this: https://docs.docker.com/get-started/

**Comment by mohsinskynet** [comment]
where to find nginx file to change the CACHE_MEM_SIZE, CACHE_DISK_SIZE, CACHE_MAX_AGE
and how to run Game Content Cache Hostnames please post a guide or tell here ty

**Comment by mintopia** [comment]
Issue has been waiting on a response for 4 months, closing.

@mohsinskynet Please raise a new issue for your problem.
